app/src/js/components/Collections/overview.js

```diff
@@ -56,7 +56,7 @@ const breadcrumbConfig = [
   },
   {
     label: 'Collections',
-    href: '/collections',
+    href: '/collections/all',
```

Comment:
Changed to /collections/all because /collections redirects to /collections/all and all query params were stored for /collections/all, but for s to /collections params were not appended.